en-US/skins/skin.ini.md

```diff
@@ -323,19 +323,25 @@ This section is to manipulate the elements for the Keys game mode including 4K a
-|      JudgeColorGreat      | RGB Color(255,255,255) |   86,254,110   | The color of the `Great` counter overlay      |
-|      JudgeColorGood       | RGB Color(255,255,255) |   0,209,255    | The color of the `Good` counter overlay       |
-|      JudgeColorOkay       | RGB Color(255,255,255) |  217,107,206   | The color of the `Okay` counter overlay       |
-|      JudgeColorMiss       | RGB Color(255,255,255) |   249,100,93   | The color of the `Miss` counter overlay       |
+###### Affected Textures: [judgement-overlay.png](/docs/skins/gameplayinterface#judgement-overlay), [judgement-overlay-background-{marv, perf, great, good, okay, miss}.png](/docs/skins/gameplayinterface#judgement-overlay-background)
+
+|            Value            |       Data Type        | Alignment | Default Values | Notes                                                                                                           |
+| :-------------------------: | :--------------------: | :-------: | :------------: | --------------------------------------------------------------------------------------------------------------- |
+|    JudgementCounterAlpha    |    Float (0.0-1.0)     |     -     |       1        | The alpha/opacity of the judgement counter                                                                      |
+|  JudgementCounterFontColor  | RGB Color(255,255,255) |     -     |  255,255,255   | The color of the judgement counter font                                                                         |
+|    JudgementCounterSize     |        Integer         |     -     |       40       | The width and height of the judgement counter                                                                   |
+|       JudgeColorMarv        | RGB Color(255,255,255) |     -     |  255,255,255   | The color of the `Marvelous` counter overlay                                                                    |
+|       JudgeColorPerf        | RGB Color(255,255,255) |     -     |  255,231,107   | The color of the `Perfect` counter overlay                                                                      |
+|       JudgeColorGreat       | RGB Color(255,255,255) |     -     |   86,254,110   | The color of the `Great` counter overlay                                                                        |
+|       JudgeColorGood        | RGB Color(255,255,255) |     -     |   0,209,255    | The color of the `Good` counter overlay                                                                         |
+|       JudgeColorOkay        | RGB Color(255,255,255) |     -     |  217,107,206   | The color of the `Okay` counter overlay                                                                         |
+|       JudgeColorMiss        | RGB Color(255,255,255) |     -     |   249,100,93   | The color of the `Miss` counter overlay                                                                         |
```

Comment:
JudgeColor also affects hitlighting tint if enabled in settings, and hiterror.

Reply:
I added an additional note below the table of the judgement counter skin.ini values regarding JudgeColor affecting hitlighting and hiterror instead of adding it one by one in the notes in the table because it'd be too long.